src/uct/tcp/tcp_iface.c

```diff
@@ -32,10 +32,14 @@ static ucs_config_field_t uct_tcp_iface_config_table[] = {
    "option usually provides better performance",
    ucs_offsetof(uct_tcp_iface_config_t, sockopt_nodelay), UCS_CONFIG_TYPE_BOOL},
 
-  {"SNDBUF", "64k",
-   "Socket send buffer size.",
+  {"SNDBUF", "32k",
+   "Socket send buffer size. 0 - default value defined by system",
    ucs_offsetof(uct_tcp_iface_config_t, sockopt_sndbuf), UCS_CONFIG_TYPE_MEMUNITS},
 
+  {"RCVBUF", "0",
```

Comment:
do you intend to keep system default value or set it to 32K (as per PR description)?